master/reference/calicoctl/resources/profile.md

```diff
@@ -2,116 +2,87 @@
+| source      | Source match parameters. |  | [EntityRule](#entityrule) | |
+| destination | Destination match parameters. |  | [EntityRule](#entityrule) | |
 
-| name | description                  | requirements         | schema  |
-|------|------------------------------|----------------------|---------|
-| type | Positive match on ICMP type. | Can be integer 1-255 | integer |
-| code | Positive match on ICMP code. | Can be integer 1-255 | integer |
+#### ICMP
 
+| Field       | Description                 | Accepted Values   | Schema | Default    |
+|-------------|-----------------------------|-------------------|--------|------------|
+| type | Match on ICMP type. | Can be integer 1-255 | integer |
+| code | Match on ICMP code. | Can be integer 1-255 | integer |
 
 #### EntityRule
 
-| name        | description                                | requirements                           | schema                        |
-|-------------|--------------------------------------------|----------------------------------------|-------------------------------|
-| tag         | Match expression on tags.                  |                                        | string                        |
-| net         | Match on CIDR.                             |                                        | string representation of cidr |
-| selector    | Selector expression.                       |  | string |
-| ports       | Restricts the rule to only apply to traffic that has a port that matches one of these ranges/values. | A list of integers and/or strings, where strings can represent a range of ports by joining the range by a colon, e.g. `'1000:2000'` | list of strings and/or integers. |
-| notTag | Negative match on tag. |  | string |
-| notNet | Negative match on CIDR. | | string representation of cidr |
-| notSelector | Negative match on selector expression. | | string |
-| notPorts      | Negative match on ports. | A list of integers and/or strings, where strings can represent a range of ports by joining the range by a colon, e.g. `'1000:2000'` | list of strings and/or integers. |
+| Field       | Description                 | Accepted Values   | Schema | Default    |
+|-------------|-----------------------------|-------------------|--------|------------|
+| tag (deprecated)      | Positive match on tag. |  | string |
+| notTag (deprecated)   | Negative match on tag. |  | string |
+| net    | Match on CIDR. | Valid IPv4 or IPv6 CIDR  | cidr |
+| notNet | Negative match on CIDR. | Valid IPv4 or IPv6 CIDR | cidr |
+| selector    | Positive match on selected endpoints. | Valid selector | [selector](#selector) |
+| notSelector | Negative match on selected endpoints. | Valid selector | [selector](#selector) |
+| ports | Positive match on the specified ports | A list of exact ports or port rages, e.g '1000:2000' | list |
+| notPorts | Negative match on the specified ports | A list of exact ports or port rages, e.g '1000:2000' | list |
```

Comment:
I don't think this explains what the format is anymore - it's not clear that it can be a number or a : separate port range.